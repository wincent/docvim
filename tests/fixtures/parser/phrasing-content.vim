""
" We want to make sure that stuff immediately following phrasing content.
" # Gets recognized
" - Appropriately
" ```
" code
" ```
" ## Even if we don't expect
" `this` kind of thing to happen very often.
" @indent
" indented content
" @dedent
" ---
" > end of special stuff
" back to normal phrasing content again.
