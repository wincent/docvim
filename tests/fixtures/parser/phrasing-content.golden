Unit
  [ DocBlock
      [ Paragraph
          [ Plaintext "We"
          , Whitespace
          , Plaintext "want"
          , Whitespace
          , Plaintext "to"
          , Whitespace
          , Plaintext "make"
          , Whitespace
          , Plaintext "sure"
          , Whitespace
          , Plaintext "that"
          , Whitespace
          , Plaintext "stuff"
          , Whitespace
          , Plaintext "immediately"
          , Whitespace
          , Plaintext "following"
          , Whitespace
          , Plaintext "phrasing"
          , Whitespace
          , Plaintext "content."
          ]
      , HeadingAnnotation "Gets recognized"
      , List [ ListItem [ Plaintext "Appropriately" ] ]
      , Fenced [ "code" ]
      , SubheadingAnnotation "Even if we don't expect"
      , Paragraph
          [ Code "this"
          , Whitespace
          , Plaintext "kind"
          , Whitespace
          , Plaintext "of"
          , Whitespace
          , Plaintext "thing"
          , Whitespace
          , Plaintext "to"
          , Whitespace
          , Plaintext "happen"
          , Whitespace
          , Plaintext "very"
          , Whitespace
          , Plaintext "often."
          ]
      , IndentAnnotation
      , Paragraph
          [ Plaintext "indented" , Whitespace , Plaintext "content" ]
      , DedentAnnotation
      , Separator
      , Blockquote
          [ Paragraph
              [ Plaintext "end"
              , Whitespace
              , Plaintext "of"
              , Whitespace
              , Plaintext "special"
              , Whitespace
              , Plaintext "stuff"
              ]
          ]
      , Paragraph
          [ Plaintext "back"
          , Whitespace
          , Plaintext "to"
          , Whitespace
          , Plaintext "normal"
          , Whitespace
          , Plaintext "phrasing"
          , Whitespace
          , Plaintext "content"
          , Whitespace
          , Plaintext "again."
          ]
      ]
  ]
